### ISSUE TITLE
eCH Standards

### ISSUE THREAD

**Issue by giacomociti** [schema-ld-admin/schema#3]
Hello, in the context of the Agroscope WebGRUD project, we're creating a couple of term sets about chemicals. One is more generic (elements of the periodic table), and the other is more specific (some compounds of agronomic interest). In both cases, we have to define properties (like atomic number and canonical smiles) and we think [eCH Standards](https://schema.ld.admin.ch/office/ech) is the right place to add them. Any objections or alternative suggestions @l00mi ?

**Comment by l00mi** [comment]
That is a topic which pops-up more frequently lately, I know that @ortnever is also interessted in a solution here. (@ortnever do yo have a link to the issue for this?)

We did check once quickly and my proposition for this would be to use a standard external dataset (most probably https://www.ebi.ac.uk/chembl/) and to create an internal "SharedDimension" based on this.

**Comment by giacomociti** [comment]
our [current ](https://int.cube-creator.lindas.admin.ch/app/shared-dimensions/https:!!!!ld.admin.ch!!vocabulary!!chemicalElements) solution is based on Wikidata

**Comment by l00mi** [comment]
Yes true, as it also links to Chembl. But then I did probably not understand the question? 

You want to describe the created new resource created from Wikidata?

**Comment by giacomociti** [comment]
we are only wondering which  'folder' to use [here](https://schema.ld.admin.ch/) to add a few 'custom' terms 
With @bergos and @dabo505 we chose eCH Standards

**Comment by l00mi** [comment]
No eCH standards is for standards which are translated from https://www.ech.ch/ to LD. Go with "common".
